pkg/model/keycloak_deployment.go

```diff
@@ -157,8 +158,9 @@ func KeycloakDeployment(cr *v1alpha1.Keycloak) *v13.StatefulSet {
 								},
 							},
 							ReadinessProbe: &v1.Probe{
+								InitialDelaySeconds: 30,
```

Comment:
30 for initial delay seems a bit high. On fast machines Keycloak can start in <15 seconds.